### PR TITLE
Only wait for 60 seconds for the faitout database.

### PR DESCRIPTION
**Pull request by bowlofeggs**
Thanks to @jeremycline for the suggestion to do this.

Signed-off-by: Randy Barlow <randy@electronsweatshop.com>

### CODE REVIEW

File: bodhi/tests/server/base.py [hunk 1]
```diff
@@ -40,7 +40,7 @@
 # https://wiki.jenkins-ci.org/display/JENKINS/Building+a+software+project#Buildingasoftwareproject-below
 if os.environ.get('BUILD_ID'):
     try:
-        req = requests.get('%s/new' % FAITOUT)
+        req = requests.get('%s/new' % FAITOUT, timeout=60)
```
Comment (bowlofeggs) [inline bodhi/tests/server/base.py:43]: I presumed that the unit here is seconds, but the requests docs do not say:

http://docs.python-requests.org/en/master/api/#requests.request

Also, 60 seconds is an arbitrary value I pulled out of nowhere.

Comment (bowlofeggs) [reply bodhi/tests/server/base.py:43]: Ah, it does seem to be seconds:

http://docs.python-requests.org/en/master/user/advanced/#timeouts

**Review by jeremycline — APPROVED** [review]
Perfect :shipit:

**Comment by pypingou** [comment]
Definitely going to re-use that in some places :D

**Pull request merged**